### PR TITLE
Add info about browsers in selective mode

### PR DESCRIPTION
**Pull request by puglieri**

### CODE REVIEW

File: docs/adguard-vpn-for-ios/overview.md [hunk 1]
```diff
@@ -43,6 +43,15 @@ You can add domains (e.g., `google.com`) or subdomains (e.g., `*.google.com`) of
 
 ![Exclusions *mobile_border](https://cdn.adguardvpn.com/content/kb/vpn/ios/2.2/exclusions-add.png)
 
+:::info Browsers in Selective mode
+
+Starting with the version 2.4, when switching to Selective mode, all browsers on the system are automatically added to the exceptions list for more convenience. This change applies under the following conditions:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-ios/overview.md:48]: we don't have the "exceptions list". it's called Exclusions list: https://crowdin.com/editor/adguard-applications/all/en-ru?view=comfortable&filter=basic&value=0#q=exclusions%20list

File: docs/adguard-vpn-for-ios/overview.md [hunk 2]
```diff
@@ -43,6 +43,15 @@ You can add domains (e.g., `google.com`) or subdomains (e.g., `*.google.com`) of
...
+
+- The user installed VPN for the first time
+- The user did not modify selector mode settings prior to v2.4
```
Comment (el-termikael) [inline docs/adguard-vpn-for-ios/overview.md:51]: we neither have "selector mode"...

File: docs/adguard-vpn-for-mac/overview.md [hunk 3]
```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
 
 Exclusion list can be easily configured. If you've added a domain and some of its subdomains, they'll be grouped inside the root one. When adding a root domain (`example.com`) its mask is added too (`*.example.com`).
 
+:::info Browsers in Selective mode
+
+Starting with the version 2.4, when switching to Selective mode, all browsers on the system are automatically added to the exceptions list for more convenience. This change applies under the following conditions:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md:45]: AdGuard VPN for iOS is the only VPN product where there still are Regular and Selective mode. In other products, both apps and extensions, it's called differently. Please check that before writing

File: docs/adguard-vpn-for-mac/overview.md [hunk 4]
```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
...
+
+- The user installed VPN for the first time
+- The user did not modify selector mode settings prior to v2.4
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md:48]: Why selector...

File: docs/adguard-vpn-for-windows/overview.md [hunk 5]
```diff
@@ -43,6 +43,15 @@ When adding domains manually, you should take into account certain details. For
 
 We recommend using the **From the list** option. Websites are grouped into eight categories: Social networks, Messengers, Video and Music streaming services, Games, Shopping, Search engines, and Work communication tools. We have placed the most popular services there, including all domain names and subdomains related to each platform.
 
+:::info Browsers in Selective mode
+
+Starting with the version 2.4, when switching to Selective mode, all browsers on the system are automatically added to the exceptions list for more convenience. This change applies under the following conditions:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/overview.md:48]: same here

File: docs/adguard-vpn-for-windows/overview.md [hunk 6]
```diff
@@ -43,6 +43,15 @@ When adding domains manually, you should take into account certain details. For
...
+
+- The user installed VPN for the first time
+- The user did not modify selector mode settings prior to v2.4
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/overview.md:51]: and here

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-mac/overview.md [hunk 7]
```diff
@@ -40,15 +40,6 @@ Also you can not only add websites to Exclusions but also choose among the lists
 
 Exclusion list can be easily configured. If you've added a domain and some of its subdomains, they'll be grouped inside the root one. When adding a root domain (`example.com`) its mask is added too (`*.example.com`).
 
-:::info Browsers in Selective mode
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md]: you shouldn't remove these blocks — the task was to mention this change for all apps. desktop ones are no less important

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-mac/overview.md (hunk 7 above)
Comment (puglieri) [reply docs/adguard-vpn-for-mac/overview.md]: Wait, I'm a bit confused. 

The task is to mention that switching to Selective mode will add all browsers to exclusions. You've mentioned in another comment that the VPN for iOS is the only product in which there are still a Regular and a Selective mode, is it right? That's why I've removed the other blocks.

Comment (el-termikael) [reply docs/adguard-vpn-for-mac/overview.md]: it's called differently, here is a screenshot:
Снимок экрана 2024-08-12 в 18 51 21
you should always check how features are named in the apps

Comment (el-termikael) [reply docs/adguard-vpn-for-mac/overview.md]: Снимок экрана 2024-08-12 в 18 52 28
now you can switch between these two modes (works the same for app exclusions)

**Review by Aydinv13 — APPROVED** [review]

File: docs/adguard-vpn-for-mac/overview.md [hunk 8]
```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
 
 Exclusion list can be easily configured. If you've added a domain and some of its subdomains, they'll be grouped inside the root one. When adding a root domain (`example.com`) its mask is added too (`*.example.com`).
 
+:::info Browsers in the exclusions list
+
+Starting with the version 2.4, all browsers on the system are automatically added to the exclusions list when the VPN is active only for selected apps and websites. This change brings more convenience for users who were often confused and didn’t know whether or not to consider their browsers as apps. It applies under the following conditions:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md:45]: Starting with the version 2.4, all browsers are automatically added to the list of exclusions when the VPN is active only for selected apps and websites. This change brings more convenience for users who were often confused and didn’t know whether or not to consider their browsers as apps or not. It applies under the following conditions:

File: docs/adguard-vpn-for-mac/overview.md [hunk 9]
```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
...
+
+- The user installed the VPN for the first time
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md:47]: The user has installed AdGuard VPN for the first time

File: docs/adguard-vpn-for-mac/overview.md [hunk 10]
```diff
@@ -40,6 +40,15 @@ Also you can not only add websites to Exclusions but also choose among the lists
 
 Exclusion list can be easily configured. If you've added a domain and some of its subdomains, they'll be grouped inside the root one. When adding a root domain (`example.com`) its mask is added too (`*.example.com`).
 
+:::info Browsers in the exclusions list
```
Comment (el-termikael) [inline docs/adguard-vpn-for-mac/overview.md:43]: :::info Browsers added to Exclusions

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Comment by el-termikael** [comment]
closes #188

**Review by el-termikael — APPROVED** [review]

**Comment by el-termikael** [comment]
don't deploy KB before v2.4 is released

**Pull request merged**